backend/clickhouse/logs.go

```diff
@@ -65,17 +70,25 @@ func (client *Client) ReadLogs(ctx context.Context, projectID int) ([]*modelInpu
 
 	for rows.Next() {
 		var (
-			Timestamp    time.Time
-			SeverityText string
-			Body         string
+			Timestamp     time.Time
+			SeverityText  string
+			Body          string
+			LogAttributes map[string]string
 		)
-		if err := rows.Scan(&Timestamp, &SeverityText, &Body); err != nil {
+		if err := rows.Scan(&Timestamp, &SeverityText, &Body, &LogAttributes); err != nil {
 			return nil, err
 		}
+
+		gqlLogAttributes := make(map[string]interface{}, len(LogAttributes))
```

Comment:
This is to match the gqlgen `Map` [scalar](https://gqlgen.com/reference/scalars/#map).